app/pages/notifications.spec.js

```diff
@@ -30,18 +32,20 @@ describe('Notifications', function() {
     });
 
     it('will notify when no notifications present', function() {
+      const stub = sinon.stub(Notifications.prototype, 'componentWillMount');
       wrapper = mount(<Notifications user={{ id: 1 }} />);
       assert(wrapper.contains(<span>You have no notifications.</span>));
+      stub.restore();
     });
   });
 
   describe('it correctly display projects', function() {
     beforeEach(function () {
       wrapper = shallow(
-        <Notifications user={{ id: 1 }} />,
```

Comment:
asynchronous calls to a remote server should go in `componentDidMount`: https://facebook.github.io/react/docs/react-component.html#componentdidmount